nuxt.config.js

```diff
@@ -23,4 +24,100 @@ c.dir = {
   store:      path.posix.join(SHELL, 'store'),
 };
 
+c.build.extend = (config, { isClient, isDev }) => {
+  // Override the webpack target, so that we get the correct mix of
+  // electron (chrome) + nodejs modules (for ipcRenderer).
+  config.target = 'electron-renderer';
+  // // Set a resolver alias for `./@` so that we can load things from @ in CSS
+  config.resolve.alias['./@'] = path.resolve(__dirname, 'pkg', 'rancher-desktop');
```

Comment:
Is it correct to still handle the '@/' alias, or do we want any remaining instances of that to trigger an error?

Reply:
Good point, I have removed the `@` alias for now.